README.md

```diff
@@ -78,16 +67,65 @@ pnpm install
   cd src && cp config-examples/* config && for i in config/*-example.js; do mv -- "$i" "${i%-example.js}.js"; done
   ```
 
-4. Set the app-config.js values for:
+2. Set the app-config.js values for:
 
 - `orsApiKey` - ORS API key to be used when ot running the app from localhost or ors valid domains
 - `bitlyApiKey` - the Bitly key (used to shorten the share URL)
 - `bitlyLogin` - the Bitly login (used to shorten the share URL)
 
-5. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
+3. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
 
 The filters, theme and hooks of the map client can be customized if needed.
 
+##### Run with `Docker`
+
+Run the app in `docker`:
+
+```shell
+# Build the image
+docker build --tag ors-map-client:local .
+
+# Run a container
+docker run -d -it -v "$(pwd)"/nginx/logs:/var/log/nginx:rw -p 127.0.0.1:8080:80 --name ors-map-client ors-map-client:local
+
+# Check the logs with
+docker logs --follow ors-map-client
+```
+
+The app should now be running at http://127.0.0.1:8080.
+
+##### Run with `docker compose`
+
+Run the app in `docker compose`:
+
+```shell
+# Build and run the image
+docker-compose up -d
+
+# Check the logs with
+docker-compose logs -ft
```

Comment:
just for consistency with Header, either use `docker-compose` only or `docker compose` only.